lib/pacemaker/Makefile.am

```diff
@@ -26,7 +26,10 @@ libpacemaker_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
 libpacemaker_la_LIBADD	= $(top_builddir)/lib/pengine/libpe_status.la \
 			$(top_builddir)/lib/cib/libcib.la \
 			$(top_builddir)/lib/lrmd/liblrmd.la \
-			$(top_builddir)/lib/common/libcrmcommon.la
+			$(top_builddir)/lib/common/libcrmcommon.la \
+			$(top_builddir)/lib/fencing/libstonithd.la \
+			$(top_builddir)/lib/services/libcrmservice.la
```

Comment:
The pc file order shouldn't matter, but LIBADD (sorta) does, from highest level to lowest level. libstonithd and libcrmservice should be between liblrmd and libcrmcommon

Reply:
Updated.